harmony_service_lib/adapter.py

```diff
@@ -181,9 +181,13 @@ def _process_catalog_recursive(self, catalog):
         result.clear_items()
         source = None
         for item in items:
+            cloned_item = item.clone()
+            # if there is a bbox, but no geometry, create a geometry from the bbox
+            if cloned_item.bbox and not cloned_item.geometry:
```

Comment:
Do we know why an item could have bbox, but no geometry?

Reply:
Harmony is creating them that way. Up until recently this didn't matter, but pystac started enforcing the spec, so we decided to add the geometry by duplicating the bbox as a polygon.